pscript/stdlib.py

```diff
@@ -305,7 +305,8 @@ def get_all_std_names():
 
 FUNCTIONS['round'] = 'Math.round // nargs: 1'
 
-FUNCTIONS['int'] = """function (x) { // nargs: 1
+FUNCTIONS['int'] = """function (x, base) { // nargs: 1 2
+    if(base !== undefined) return parseInt(x, base);
     return x<0 ? Math.ceil(x): Math.floor(x);
```

Comment:
Actually, I think we can do `FUNCTIONS['int'] = "parseInt // nargs: 1 2"`

Reply:
I didn't think of that .) Though parseInt may be slower as people [said](https://teamtreehouse.com/community/wont-parseint-do-the-same-thing-as-mathfloor-but-faster-since-you-dont-have-to-call-up-the-math-class) in 2015. (benchmark is not available anymore)

Reply:
Good call. benchmarking shows that this is still the case (at least on FF). Let's keep it as is then :)